packages/opentelemetry-metrics/src/Handle.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+
+/**
+ * CounterHandle is an implementation of the {@link CounterHandle} interface.
```

Comment:
This comment in itself doesn't seem too useful. I'm OK with removing it if needed. (Same applies for comment below)

I guess I was thinking this would instead talk a bit about what a counter is and what a handle is, but maybe the expectation doesn't make sense - perhaps people should just be reading the READMEs anyway.